assets/src/data/content/model/elements/types.ts

```diff
@@ -78,6 +86,23 @@ export interface ImageInline extends BaseImage {
   type: 'img_inline';
 }
 
+interface FormulaChildren<typeIdentifier>
```

Comment:
Separated out the formula with children (richtext) vs. the formula with a source attribute (mathml / latex) so I could type it that they weren't both allowed at the same time.